.ci/pitest.sh

```diff
@@ -47,6 +46,15 @@ pitest-annotation|pitest-design \
   checkPitestReport "${ignoredItems[@]}"
   ;;
 
+# Until Issue #9351 is fixed
```

Comment:
please change comment to make it possible to check it by job, like

`till #9351`

Reply:
Done